ai_flow/common/util/db_util/session.py

```diff
@@ -77,6 +86,14 @@ def create_session(db_uri=None):
     return session_factory()
 
 
+def new_session(db_uri=None):
```

Comment:
What's the purpose of having this function?

Reply:
because create_session function return a context manager.